### ISSUE TITLE
Windows GUI

### ISSUE THREAD

**Issue by nyxtryx** [nyxtryx/Mouse-Mover#1]
## Summary

A GUI to allow users to start, and stop, the background script - effectively pausing the program when they wish.

### Basic example
This could be used for breaks, so that users are correctly shown as idle on breaks rather than appearing to be still present.

### Motivation

While this is currently possible by simply closing the script, the steps to restart the script are cumbersome in some enterprise environments - a GUI would simplify this.

**Comment by nyxtryx** [comment]
My plan is to use something like tkinter for the GUI - https://docs.python.org/3/library/tkinter.html (a more directed page for people newer to Python - https://realpython.com/python-gui-tkinter/)

**Comment by joaojesus81** [comment]
@nyxtryx i have a little bit of experience with creating python UI's both with tkinter and pyside6 (you can check my repos for examples) and would be happy to have a shot at creating a simple UI for this project if you'd like me to have a go. :) Let me know

**Comment by nyxtryx** [comment]
@joaojesus81 by all means!

I won't have much time to work on much more than I've done because of work and uni, so any help is apprecaited 🙂

**Comment by joaojesus81** [comment]
@nyxtryx brilliant. I'll start with something simple and then take it from there. Any problem if i used pyside6 instead of tkinter? Even thought it would probably be enough, tkinter frustrates me quite a bit.

**Comment by nyxtryx** [comment]
@joaojesus81 not at all! As they say "beggars can't be choosers", and it gives me an excuse to learn something new 😁

My idea was *very* simple though - literally a "start" and "stop" button to set the run variable to true or false 😅

But I'm definitely open to better ideas/features
